src/server/JasmineGraphServer.cpp

```diff
@@ -1994,31 +1994,53 @@ void JasmineGraphServer::initiateOrgCommunication(std::string graphID, std::stri
 
     Utils utils;
     int fl_clients = stoi(utils.getJasmineGraphProperty("org.jasminegraph.fl_clients"));
-        std::string flagPath = utils.getJasmineGraphProperty("org.jasminegraph.fl.flag.file");
-    int threadLimit = fl_clients + 1;
+    int orgs_count = stoi(utils.getJasmineGraphProperty("org.jasminegraph.fl.num.orgs"));
+    std::string flagPath = utils.getJasmineGraphProperty("org.jasminegraph.fl.flag.file");
+    int threadLimit = fl_clients+1;
+    int org_thread_count = 0;
     std::thread *workerThreads = new std::thread[threadLimit];
 
     utils.editFlagOne(flagPath);
     Utils::worker workerInstance;
     vector<Utils::worker> workerVector = utils.getWorkerList(sqlite);
 
+    std::thread *trainThreads = new std::thread[orgs_count];
+
     if (utils.getJasmineGraphProperty("org.jasminegraph.fl.aggregator") == "true"){
    
         std::thread *orgAggThread = new std::thread[1];
         orgAggThread[0] = std::thread(initiateAggregator,"localhost",  stoi(workerVector[0].port), 
                                         stoi(workerVector[0].dataPort),trainingArgs,fl_clients, "1");
         orgAggThread[0].join();
+
+    }
+
+    std::ifstream file(utils.getJasmineGraphProperty("org.jasminegraph.fl.organization.file"));
+    if (file.is_open()) {
+        std::string line;
+        while (std::getline(file, line)) {
+            line = utils.trim_copy(line, " \f\n\r\t\v");
+            std::vector<std::string> strArr = Utils::split(line.c_str(), '|');
+            std::string host = strArr[0].c_str();
+            int port = stoi(strArr[1].c_str());
+            std::string trainingArgs = "--graph_id " + strArr[2];
+            server_logger.log(to_string(strArr.size()), "info");
```

Comment:
This log message does not have any meaning

Reply:
Removed in 58678b1a64d640daa8361e0ff3620faf4205adb9

Reply:
Removed in 58678b1a64d640daa8361e0ff3620faf4205adb9

Reply:
Done